skafka/src/main/scala/com/evolutiongaming/skafka/consumer/Consumer.scala

```diff
@@ -34,9 +34,14 @@ trait Consumer[F[_], K, V] {
 
   def assignment: F[Set[TopicPartition]]
 
+  def subscribe(topics: Nes[Topic], listener: RebalanceListener1[F]): F[Unit]
```

Comment:
please addd alternative methods

```
def subscribe(topics: Nes[Topic]): F[Unit]

def subscribe(topics: Nes[Topic]): F[Unit]
```

as you've deprecated the accepting optional `RebalanceListener`

Reply:
thanks, will do

Reply:
fixed in 58c64e7d